app/models/admin_user.rb

```diff
@@ -6,6 +6,5 @@ class AdminUser < ActiveRecord::Base
          :recoverable, :rememberable, :trackable, :validatable
 
   # Setup accessible (or protected) attributes for your model
-  attr_accessible :email, :password, :password_confirmation, :remember_me
```

Comment:
Oh true, that's great then. There wasn't any point to `attr_accessible` to begin with then...